DOCS-4139 Service Catalog Setup Update (#15407)
DataDog/documentation: content/en/tracing/service_catalog/setup.md

To discover other services through existing Datadog telemetry such as infrastructure metrics, go to the **Discover Services** tab and follow instructions there. Discovery uses the `DD_SERVICE` [Unified Service Tagging convention][4] to locate services sending data to your Datadog organization.

> ## Set up the GitHub Apps integration

Reviewer: ```suggestion
## Store and edit service definitions in GitHub
```

> You can use the [GitHub Apps integration][5] to display links for service definitions in GitHub.

Reviewer: ```suggestion
Configure the [GitHub Apps integration][5] so that you can directly link from where you view the service's definition in Service Catalog to where it's stored and editable in GitHub.
```

To install the GitHub Apps integration, navigate to the [integration tile][6] and click **Link GitHub Account** in the **Repo Configuration** tab. 

### Service definition YAML files

Datadog scans for the `service.datadog.yaml` file at the root of each repository with read permissions. You can register multiple services in one YAML file by creating multiple YAML documents. Separate each document with three dashes (`---`).

### Modify service definition

> The **Edit in Github** button appears in the **Definition** tab and links users directly to a pre-templated commit that you can modify.

Reviewer: ```suggestion
When this integration with GitHub has been set up for your service definitions, an **Edit in Github** button appears in the service's **Definition** tab and links you to GitHub to commit changes.
```

> When the GitHub Apps integration has been set up for your service definitions, an **Edit in Github** button appears in the service's **Definition** tab and links you to GitHub to commit changes.

Reviewer: ```suggestion
When the GitHub Apps integration is set up for your service definitions, an **Edit in Github** button appears in the service's **Definition** tab and links you to GitHub to commit changes.
```

> {{< img src="tracing/service_catalog/svc_cat_contextual_link.png" alt="An Edit in Github button appears in the Definition tab of a service in the Service Catalog" style="width:90%;" >}}

Reviewer: this screen cap shows too much UI to easily locate the button described. Maybe crop a bit or circle the button (or both).

> Once you have updated the YAML files for your repositories, your changes propagate to the Service Catalog. To prevent accidental overwriting, Datadog recommends creating and modifying your service definition files with the GitHub Apps integration or the [Service Definition API endpoints][1].

Reviewer: ```suggestion
After you update the YAML files for your repositories, your changes propagate to the Service Catalog. To prevent accidental overwriting, create and modify your service definition files with the GitHub Apps integration or the [Service Definition API endpoints][1].
```
